scripts/env/local.sh

```
@@ -0,0 +1,295 @@
+    docker compose -f docker-compose.yml up -d dev
+    
+    # Wait for server to be ready
+    log_info "Waiting for dev server to be ready on port 3000..."
+    for i in {1..30}; do
+        if curl -f http://localhost:3000 >/dev/null 2>&1; then
+            log_success "Dev server is ready!"
+            log_info "🚀 Development server running at http://localhost:3000"
+            break
+        fi
+        sleep 2
+        if [ $i -eq 30 ]; then
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)